src/node_api.h

```diff
@@ -166,6 +170,18 @@ NAPI_EXTERN napi_status napi_get_value_string_length(napi_env env,
                                                      napi_value value,
                                                      int* result);
 
+// Gets the number of BYTES in the LATIN-1 encoded representation of the string.
+NAPI_EXTERN napi_status napi_get_value_string_latin1_length(napi_env env,
```

Comment:
It seems to me that napi_get_string_utf16_length is calling String::Length(). So is napi_get_string_utf16_length also redundant?